mathics_django/doc/utils.py

```diff
@@ -122,6 +122,19 @@ def repl_list(match):
         def repl_hypertext(match):
             tag = match.group("tag")
             content = match.group("content")
+            #
```

Comment:
@rocky , please check if this comment is OK.

Reply:
@mmatera Excellent - thanks!